test/common/schemaform/widgets/SelectWidget.unit.spec.jsx

```diff
@@ -122,4 +122,24 @@ describe('Schemaform <SelectWidget>', () => {
     });
     expect(onBlur.calledWith('testing', 'test')).to.be.true;
   });
+  it('should not render blank option when default exists', () => {
```

Comment:
Should there be the inverse test? Should it render a blank option when no default exists? Is that a bit redundant?